apps/userdashboard/serializers.py

```diff
@@ -0,0 +1,144 @@
+from django.template import defaultfilters
+from django.utils import timezone
+from django.utils.encoding import force_str
+from django.utils.hashable import make_hashable
+from django.utils.translation import gettext as _
+from easy_thumbnails.files import get_thumbnailer
+from rest_framework import serializers
+
+from adhocracy4.comments.models import Comment
+from apps.classifications.models import CLASSIFICATION_CHOICES
+from apps.moderatorfeedback.serializers import \
+    ModeratorCommentStatementSerializer
+
+
+class ModerationCommentSerializer(serializers.ModelSerializer):
+
+    category_counts = serializers.SerializerMethodField()
+    comment_url = serializers.SerializerMethodField()
+    has_pending_notifications = serializers.SerializerMethodField()
+    last_edit = serializers.SerializerMethodField()
+    time_of_last_notification = serializers.SerializerMethodField()
+    user_name = serializers.SerializerMethodField()
+    user_image = serializers.SerializerMethodField()
+    user_profile_url = serializers.SerializerMethodField()
+    moderator_statement = ModeratorCommentStatementSerializer(read_only=True)
+
+    class Meta:
+        model = Comment
+        fields = ['category_counts', 'comment', 'comment_url',
+                  'last_edit', 'pk', 'has_pending_notifications', 'is_blocked',
+                  'is_moderator_marked', 'moderator_statement',
+                  'time_of_last_notification', 'user_image', 'user_name',
+                  'user_profile_url']
+
+    # helper
+    def has_pending_and_archived_notifications(self, comment):
```

Comment:
I think if we only used internally, there is the convention to call it `_has_pending_and_archived_notifications`.

Reply:
Same for the other helpers.

Reply:
Oh yes, good point, will change that!